websites/0-9/1Shows/presence.ts

```diff
@@ -0,0 +1,194 @@
+
+          const ratingElement = document.querySelector('.radial-progress span.text-white')
+          const rating = ratingElement?.textContent?.trim() || 'N/A'
+
+          const runtimeElement = document.querySelector('#Movie\\ Runtime time p')
+
+          const runtime = runtimeElement?.textContent?.match(/\d+/)?.[0] || 'N/A'
+
+          const releaseDateElement = document.querySelector('#Movie\\ Release\\ Date time p')
+
+          let releaseDate = releaseDateElement?.textContent?.trim() || 'N/A'
+
+          if (releaseDate !== 'N/A') {
+            const dateParts = releaseDate.split(', ')
+            if (dateParts.length === 3) {
+              releaseDate = `${dateParts[1]} ${dateParts[2]}`
+            }
+          }
+
+          presenceData.state = `⭐ ${rating} 🕒 ${runtime} mins 🗓️ ${releaseDate}`
+
+          const posterElement = document.querySelector('figure img.object-cover')
+
+          const posterSrc = posterElement?.getAttribute('src')
+
+          presenceData.largeImageKey = posterSrc
+
+          // Check URL parameter for streaming
+          const urlParams = new URLSearchParams(window.location.search)
+          const isStreaming = urlParams.get('streaming') === 'true'
+          presenceData.smallImageKey = isStreaming ? Assets.Play : Assets.Pause
+        }
+        break
+      }
+
+      default:
+        presenceData.details = 'Browsing a Movie'
+        break
+    }
+  }
+
+  if (pathname.includes('/tv/')) {
+    switch (pathname.replace(/^\/+/, '').split('/')[0]) {
+      case 'tv': {
+        const match = pathname.match(/\/tv\/(\d+)(?:-([^/]+))?/)
+
+        if (match && match[1]) {
+          const tmdbId = match[1]
+          const showName = match[2]?.replace(/-/g, ' ') || 'Unknown Show'
+
+          const formattedShowName = showName
+            .split(' ')
+            .map(word => word.charAt(0).toUpperCase() + word.slice(1))
+            .join(' ')
+
+          const watchHistory = JSON.parse(localStorage.getItem('watch-history') || '{}')
+          const showData = watchHistory[tmdbId] || {
+            last_season_watched: '1',
+            last_episode_watched: '1',
+          }
+          const seasonNo = showData.last_season_watched
+          const episodeNo = showData.last_episode_watched
+
+          presenceData.name = `Watching ${formattedShowName} S${seasonNo}E${episodeNo}`
+          presenceData.details = '1Shows.com'
+
+          const ratingElement = document.querySelector('.radial-progress span.text-white')
+          const rating = ratingElement?.textContent?.trim() || 'N/A'
+
+          const releaseDateElement = document.querySelector('#TV\\ Shows\\ Air\\ Date time')
+          let releaseDate = releaseDateElement?.textContent?.trim() || 'N/A'
+
+          if (releaseDate !== 'N/A') {
+            const dateParts = releaseDate.split(', ')
+            if (dateParts.length === 3) {
+              releaseDate = `${dateParts[1]} ${dateParts[2]}`
+            }
+          }
```

Comment:
Let's try to not added variables only used once, if they can be simplified like this.
```suggestion
          const rating = document.querySelector('.radial-progress span.text-white')?.textContent?.trim() || 'N/A'
          
          let releaseDate = document.querySelector('#TV\\ Shows\\ Air\\ Date time')?.textContent?.trim() || 'N/A'

          if (releaseDate !== 'N/A') {
            const dateParts = releaseDate.split(', ')
            if (dateParts.length === 3) {
              releaseDate = `${dateParts[1]} ${dateParts[2]}`
            }
          }
```

Reply:
Done applied changes